L1.py

```diff
@@ -0,0 +1,34 @@
+from random import *
+from copy import *
+
+def sortowanie_1(lista):
+    zakres = len(lista) - 1

```

Comment:
jeśli zakres służy Ci tylko po to by użyć go linię niżej w warunku- pisz bezpośrednio w warunku.